### ISSUE TITLE
Azure - Security Group Detect Remediate parity with AWS

### ISSUE THREAD

**Issue by justinhauer** [cloud-custodian/cloud-custodian#4957]
Does this functionality exist for Azure?

https://cloud-custodian.github.io/cloud-custodian/docs/aws/examples/securitygroupsdetectremediate.html

**Comment by aluong** [comment]
What are you trying to do in Azure? Can you be more specific?

**Comment by stefangordon** [comment]
Yep, this should just be a policy targeting the event `NetworkSecurityGroupWrite`.
